janis_bioinformatics/data_types/fasta.py

```diff
@@ -51,7 +51,7 @@ def name():
 
     @staticmethod
     def secondary_files():
-        return ["^.dict"]
+        return ["^.dict", *FastaFai.secondary_files()]
```

Comment:
We probably want to make the class inherit from FastaFai and then change the FastaWithIndexes. I've pushed a commit (af9abda0458166146dc7fe81f01bf1e3c917d2c8) to master to address your change here. I'll merge this PR after this change is incorporated.

Reply:
Hey @matthdsm, can you just cherry-pick af9abda0458166146dc7fe81f01bf1e3c917d2c8 into your branch, and then I'll merge your PR.

Reply:
I reverted the offending commit, so I suppose the merge should be good to go now.
I'll try not to mess things up anymore 😞

Reply:
Please keep fiddling around!! I’m glad you found a problem with my implementation. I have the upper hand that I know how it was built ;)